lib/bundler/rubygems_ext.rb

```diff
@@ -166,6 +166,10 @@ def matches_spec?(spec)
 
       requirement.satisfied_by?(spec.version)
     end unless allocate.respond_to?(:matches_spec?)
+
+    def force_version?
+      false
+    end
```

Comment:
we could, I'm just worried about the performance implications of that. Note that we already monkey-patch in `groups`